packages/bsky/tests/feed-generation.test.ts

```diff
@@ -582,6 +583,116 @@ describe('feed generation', () => {
       )
     })
 
+    describe('postgates', () => {
```

Comment:
thoughts on moving this into it's own test suite next to `threadgating`?

this test suite is really about appview integration with feed generators & while that does intersect to some degree with postgates, I think we might want to have a separate test suite for it as a concept

Reply:
oh wait i shoulda read further. i thought there were more tests here. think this is cozy here 👍

Reply:
I need to add some more so I think it's a good idea to split this out before this PR is said and done 👍